### PR TITLE
Avatar badge alert

### PR DESCRIPTION
**Pull request by dipteekhd**
Review UI :
https://githubbox.com/dipteekhd/KHD-UI/tree/avatar-badge-alert

### CODE REVIEW

**Review by Abhijeet43 — APPROVED** [review]
Code looks good!!!

**Review by Kevin-Solomon — APPROVED** [review]

File: components/alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        </div>
+
+    </div>
+
+    <!-- Dismiss Alert  -->
+    <div class="alert-container p-lg-all">
+
+        <div class="alert alert--success p-sm-all m-xs-all rounded-sm" role="alert">
+            <p class="text-sm semibold-font">This is a success alert</p>
+            <button class="alert--dismiss">
```
Comment (VishalPatil18) [inline components/alert/alert.html:39]: The close buttons could be given a `cursor: pointer;` so that the user knows that these are clickable buttons.

**Review by Prathyusha-96 — APPROVED** [review]

File: components/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+}
+.shadow {
+  box-shadow: var(--grey-200) 0px 1px 3px 0px, var(--grey-100) 0px 0px 0px 1px;
+}
+
+.avatar-letter-sm {
+  height: 32px;
+  width: 32px;
+  background-color: var(--grey-200);
+  font-size: 16px;
```
Comment (VishalPatil18) [inline components/avatar/avatar.css:48]: Here what you could do is as `background-color` and `font-size` is same in all the three `sm, md, lg` classes so you could create a class called `avatar-letter` and add those in that. This helps is avoiding code repetition.

**Review by VishalPatil18 — APPROVED** [review]
Just a couple of suggestions, apart from that the code looks clean. Also, I liked the naming convention you've followed. It's consistent throughout. Great Job Diptee 👍

File: components/avatar/avatar.css (hunk 2 above)
Comment (dipteekhd) [reply components/avatar/avatar.css:48]: did the required changes. Thanks for good suggestion !

**Pull request merged**